src/Builder.java

```diff
@@ -43,8 +43,7 @@ int build() {
 		run("junit5-jupiter-starter-gradle-groovy", "gradlew", "clean", "test");
 		run("junit5-jupiter-starter-gradle-kotlin", "gradlew", "clean", "test");
 		run("junit5-jupiter-starter-maven", "mvnw", "clean", "test");
-		// TODO bootstrap bazel
-		// TODO run("junit5-jupiter-starter-bazel", "bazel", "test", "//...");
+		run("junit5-jupiter-starter-bazel", "python", "bazelisk.py", "test", "//...");
```

Comment:
Oh, really?

When I run `python bazelisk.py test` on my Windows machine, it can't find any of the tests, whereas if I run `python bazelisk.py test //...`, it can then find the tests and run them.

:thinking:

Reply:
Yep! The difference between `bazel test` and `bazel test //...` is that the former will basically do all the work required to run tests (start Bazel, parse flags and config files, instantiate all classes for its runtime machinery, ...), but then doesn't actually test anything and just shuts down again. It's not an error to call Bazel like this, because it's sometimes useful when you automate Bazel via a script, in integration tests or for developers.

On the other hand `bazel test //...` means "run every single test in my WORKSPACE". (The `...` expands recursively to all packages and all targets contained in them.) This is probably what you want.

As a hint, this command will also *build* all targets that match the target pattern (so... everything in this case), even if they're not tests. By adding the flag `--build_tests_only`, Bazel will only build & test actual tests and not build anything unrelated. This can also avoid errors in case you have targets that are platform-specific, so can't be built unconditionally. If this sounds good, my recommendation would be to use: `bazel test --build_tests_only //...`. :)